### PR TITLE
Add vep

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] Added VEP as a module to annotate snvs
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/tomte _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/prepare_references.nf [hunk 1]
```diff
@@ -65,6 +67,9 @@ workflow PREPARE_REFERENCES {
         BEDTOINTERVALLIST( GET_RRNA_TRANSCRIPTS.out.bed.map {it -> [ [id:it.name], it ]}, ch_dict )
         ch_versions = ch_versions.mix(BEDTOINTERVALLIST.out.versions)
 
+        UNTAR_VEP_CACHE (ch_vep_cache)
+        ch_versions = ch_versions.mix(UNTAR_VEP_CACHE.out.versions)
```
Comment (jemten) [inline subworkflows/local/prepare_references.nf:71]: What happens when you send in a vep cache in the for of a directory and not a tarball?

File: workflows/tomte.nf
Comment (jemten) [inline workflows/tomte.nf:1]: Could you add the vep report to the multiqc channel

File: conf/modules.config [hunk 2]
```diff
@@ -302,3 +302,33 @@ process {
...
+            '--buffer_size 20000',
+            '--format vcf --max_sv_size 248956422',
+            '--appris --biotype --cache --canonical --ccds --compress_output bgzip',
+            '--domains --exclude_predicted --force_overwrite',
+            '--hgvs --humdiv --no_progress --no_stats --numbers',
+            '--merged --polyphen p --protein --offline --regulatory --sift p --symbol --tsl',
+            '--uniprot --vcf'
+        ].join(' ')
+        publishDir = [
+            path: { "${params.outdir}/annotate_snv" },
```
Comment (jemten) [inline conf/modules.config:329]:
```suggestion
            path: { "${params.outdir}/annotate_vep" },
```

**Review by jemten — COMMENTED** [review]
Nice work!
Remember to add vep to citations

File: subworkflows/local/prepare_references.nf (hunk 1 above)
Comment (Lucpen) [reply subworkflows/local/prepare_references.nf:71]: You are right! I fixed it in the config

File: workflows/tomte.nf
Comment (Lucpen) [reply workflows/tomte.nf:1]: Doen

**Review by jemten — APPROVED** [review]
Looks good!

**Pull request merged**